code/__DEFINES/reactions.dm

```diff
@@ -53,16 +53,15 @@
 #define METAL_HYDROGEN_RESEARCH_MAX_AMOUNT	3000
 #define STIMULUM_RESEARCH_AMOUNT			50
 //Plasma fusion properties
-#define FUSION_ENERGY_THRESHOLD				3e9 	//Amount of energy it takes to start a fusion reaction
 #define FUSION_MOLE_THRESHOLD				250 	//Mole count required (tritium/plasma) to start a fusion reaction
 #define FUSION_TRITIUM_CONVERSION_COEFFICIENT (1e-10)
 #define INSTABILITY_GAS_POWER_FACTOR 		0.003
 #define FUSION_TRITIUM_MOLES_USED  			1
 #define PLASMA_BINDING_ENERGY  				20000000
-#define TOROID_VOLUME_BREAKEVEN			1000
-#define FUSION_TEMPERATURE_THRESHOLD	    10000
+#define TOROID_VOLUME_BREAKEVEN				1000
+#define FUSION_TEMPERATURE_THRESHOLD	    9000
 #define PARTICLE_CHANCE_CONSTANT 			(-20000000)
-#define FUSION_RAD_MAX						2000
+#define FUSION_RAD_MAX						1500
 #define FUSION_RAD_COEFFICIENT				(-1000)
 #define FUSION_INSTABILITY_ENDOTHERMALITY   2
 #define FUSION_MAXIMUM_TEMPERATURE			1e8
```

Comment:
Basically how fusion works is that the Toroid is affected by the volume in wich the reaction is happening, an so setting up the instability. In reactions.dm i changed the size of the Toroid to be fixed for all volumes so that the instability is more consistent with all kinds of fusion reactions (pipes, canisters, open air). 
These changes are here to allow fusion to be much more consistent in its making (so the minimum heat is lower) but also i lowered the maximum rads since it got "easier" to start fusion

if you want these things explained in the changelog i'll write it there too

the FUSION_ENERGY_THRESHOLD was a leftover from fusion v5 so i just removed it

Reply:
Thanks for the explanation, I think I understand the reasoning behind it, seems kosher to me, atmos could do with a lower entry floor